src/structures/StageChannel.js

```diff
@@ -17,6 +17,15 @@ class StageChannel extends BaseGuildVoiceChannel {
     this.topic = data.topic;
   }
 
+  /**
+   * The instance of this stage channel, if it exists
+   * @type {StageInstance|undefined}
```

Comment:
Don't we usually do this?
```suggestion
   * @type {?StageInstance}
```